json2html/jsonconv.py

```diff
@@ -78,88 +78,81 @@ def columnHeadersFromListOfDicts(self, inputtedJson):
-                listMarkup = ''
-                listMarkup += '<ul><li>'
-                listMarkup += '</li><li>'.join([markup(child) for child in entry])
-                listMarkup += '</li></ul>'
-                return listMarkup
-            if (isinstance(entry, dict) == True):
-                return self.iterJson(entry)
+            if isinstance(entry, list):
+                list_markup = '<ul><li>'
+                list_markup += '</li><li>'.join([markup(child) for child in entry])
+                list_markup += '</li></ul>'
+                return list_markup
+            if isinstance(entry, dict):
+                return self.iter_json(entry)
 
             # safety: don't do recursion over anything that we don't know about
             # - iteritems() will most probably fail
             return ''
 
-        convertedOutput = ''
+        converted_output = ''
 
         global table_attributes
-        table_init_markup = "<table %s>" % (table_attributes)
-        convertedOutput = convertedOutput + table_init_markup
+        table_init_markup = "<table %s>" % table_attributes
+        converted_output += table_init_markup
 
         try:
-            for (k, v) in inputtedJson.iteritems():
-                convertedOutput = convertedOutput + '<tr>'
-                convertedOutput = convertedOutput + '<th>' + markup(k) + '</th>'
+            for (k, v) in inputted_json.iteritems():
+                converted_output += '<tr><th>' + markup(k) + '</th>'
 
-                if (v is None):
+                if v is None:
                     v = unicode("")
-                if(isinstance(v, list)):
-                    column_headers = self.columnHeadersFromListOfDicts(v)
+                if isinstance(v, list):
+                    column_headers = self.column_headers_from_list_of_dicts(v)
                     if column_headers is not None:
-                        convertedOutput = convertedOutput + '<td>'
-                        convertedOutput += table_init_markup
-                        convertedOutput += '<tr><th>' + '</th><th>'.join(column_headers) + '</th></tr>'
+                        converted_output += '<td>'
+                        converted_output += table_init_markup
+                        converted_output += '<tr><th>' + '</th><th>'.join(column_headers) + '</th></tr>'
                         for list_entry in v:
-                            convertedOutput = convertedOutput
-                            convertedOutput += '<tr><td>'
-                            convertedOutput += '</td><td>'.join([markup(list_entry[column_header]) for column_header in column_headers])
-                            convertedOutput += '</td></tr>'
+                            converted_output += '<tr><td>'
+                            converted_output += '</td><td>'.join([markup(list_entry[column_header]) for column_header in
+                                                                 column_headers])
```

Comment:
how about

    converted_output += '</td><td>'.join([
        markup(list_entry[column_header])
        for column_header in column_headers
    ])

I like this style of closing brackets better, I find it much more readable.